app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -234,13 +146,6 @@ class SmokeTest {
             }.enterURLAndEnterToBrowser(defaultWebPage.url) {
                 verifyEnhancedTrackingProtectionPanelNotVisible()
             }.openThreeDotMenu {
-            }.clickAddOnsReportSiteIssue {

```

Comment:
Not sure if this test should be enabled after the changes you did...or even though you removed part of it it is still failing...

Reply:
I removed that part because the Report site issue was removed from the menu and had to update it everywhere.